examples/powerMeter/powerMeter.ino

```diff
@@ -0,0 +1,46 @@
+/**
+ * powerMeter.ino
+ * Example implementing reading from the PZEM-014 energy meter
+ * Author: S Alexander
+ * Last Updated: 24/02/2024
+ */
+
+#include <WindsOfChange.h>
+
+// Pin connections - Change as required
+const int pin_de = 2;
+const int pin_re = 3;
+
+// Device Modbus address - Change as required
+const int device_address = 0x01;
+
+PowerMeter sensor(device_address, pin_de, pin_re);
+
+void setup() {
+  // TODOLater - Check this will work, if PowerMeter is using Serial already
```

Comment:
The `ModbusMaster` object uses `Serial` in its initialisation, so it may be unavailable for use elsewhere (hopefully not though, because if it isn't available then a _lot_ more of this library's code won't work)